docs/layout-of-source-files.rst

```diff
@@ -111,6 +111,13 @@ activate it using ``pragma experimental ABIEncoderV2;`` - we kept
 the same pragma, even though it is not considered experimental
 anymore.
 
+.. note::
+  This pragma applies to all the code defined in the file where it's activated,
```

Comment:
```suggestion
  This pragma applies to all the code defined in the file where it is activated,
```

I think we don't use contractions in docs.

Reply:
Done.